src/parsley/parsley_message.py

```
@@ -0,0 +1,49 @@
+    data: T # ParsleyDataType
+
+    @field_validator("msg_prio")
+    def validate_msg_prio(cls, value):
+        if value not in mt.msg_prio:
+            raise ValueError(f"Invalid msg_prio type '{value}'")
+        return value
+    
+    @field_validator("msg_type")
+    def validate_msg_type(cls, value):
+        if value not in mt.msg_type:
+            raise ValueError(f"Invalid msg_type type '{value}'")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)